firmware/thruna/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -269,13 +280,14 @@ class BmsStateMachineTest : public BaseStateMachineTest
     struct OkStatus *         imd_ok;
     struct OkStatus *         bspd_ok;
     struct Accumulator *      accumulator;
+    struct SocStats *         soc_stats;
     struct Airs *             airs;
     struct PrechargeRelay *   precharge_relay;
     struct TractiveSystem *   ts;
     struct Clock *            clock;
     struct Eeprom *           eeprom;
 };
-
+#include <assert.h>
```

Comment:
do you need this?

Reply:
No, not sure why that's there good catch